packages/forms/src/UIForm/UIForm.component.test.js

```diff
@@ -17,24 +18,34 @@ describe('UIForm component', () => {
 
 	it('should render form', () => {
 		// when
-		const wrapper = shallow(<UIForm {...data} {...props} />);
+		const wrapper = shallow(<UIForm {...data} {...props} i18n={i18n} />);
 
 		// then
-		expect(wrapper.getElement()).toMatchSnapshot();
+		expect(
+			wrapper
+				.find('TalendUIForm')
+				.dive()
```

Comment:
avoid using `.dive()` please.
Here you want to test the component, export it and use it here instead of the translated one.
Same for all tests in this file